src/main/java/graphql/normalized/NormalizedValuesResolver.java

```diff
@@ -0,0 +1,358 @@
+//
+//        Map<String, ObjectField> inputFieldsByName = mapObjectValueFieldsByName(inputValue);
+//
+//        List<GraphQLInputObjectField> inputFieldTypes = fieldVisibility.getFieldDefinitions(type);
+//        for (GraphQLInputObjectField inputFieldType : inputFieldTypes) {
+//
+//            GraphQLInputType fieldType = inputFieldType.getType();
+//            String fieldName = inputFieldType.getName();
+//            ObjectField field = inputFieldsByName.get(fieldName);
+//            Object defaultValue = inputFieldType.getDefaultValue();
+//            boolean hasValue = field != null;
+//            Object value;
+//            Value fieldValue = field != null ? field.getValue() : null;
+//            if (fieldValue instanceof VariableReference) {
+//                String variableName = ((VariableReference) fieldValue).getName();
+//                hasValue = coercedVariableValues.containsKey(variableName);
+//                value = coercedVariableValues.get(variableName);
+//            } else {
+//                value = fieldValue;
+//            }
+//            if (!hasValue && inputFieldType.hasSetDefaultValue()) {
+//                //TODO: default value should be coerced
+//                coercedValues.put(fieldName, new NormalizedInputValue(simplePrint(fieldType), defaultValue));
+//            } else if (isNonNull(fieldType) && (!hasValue || value == null)) {
+//                throw new NonNullableValueCoercedAsNullException(inputFieldType);
+//            } else if (hasValue) {
+//                if (value == null) {
+//                    coercedValues.put(fieldName, null);
+//                } else if (fieldValue instanceof VariableReference) {
+//                    coercedValues.put(fieldName, new NormalizedInputValue(simplePrint(fieldType), value));
+//                } else {
+//                    value = coerceValueAst(fieldVisibility, fieldType, fieldValue, coercedVariableValues);
+//                    coercedValues.put(fieldName, new NormalizedInputValue(simplePrint(fieldType), value));
+//                }
+//            } else {
+//                // nullable type && hasValue == false && hasDefaultValue == false
+//                // meaning no value was provided for this field
+//            }
+//        }
+//        return coercedValues;
+//    }
+//
+//    private void assertNonNullInputField(GraphQLInputObjectField inputTypeField) {
+//        if (isNonNull(inputTypeField.getType())) {
+//            throw new NonNullableValueCoercedAsNullException(inputTypeField);
+//        }
+//    }
+//
+//    private Map<String, ObjectField> mapObjectValueFieldsByName(ObjectValue inputValue) {
+//        Map<String, ObjectField> inputValueFieldsByName = new LinkedHashMap<>();
+//        for (ObjectField objectField : inputValue.getObjectFields()) {
+//            inputValueFieldsByName.put(objectField.getName(), objectField);
+//        }
+//        return inputValueFieldsByName;
+//    }
+//}
```

Comment:
Whats with this commented out class - is it dead or TODO code?

Reply:
dead ... will clean up